modcc/io/bulkio.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+// Read or write the contents of a file in toto.
+
+#include <string>
+#include <iterator>
+#include <fstream>
+
+namespace io {
+
+template <typename HasAssign>
+void snarf(std::istream& in, HasAssign& A) {
```

Comment:
I appreciate that `snarf` and friends have history, but I would prefer to use something more descriptive for people who have been programming less than 20 years.